src/components/LanguageBar/LanguageBar.tsx

```diff
@@ -17,8 +20,9 @@ export interface LanguageBarProps {
 export const LanguageBar: FunctionComponent<LanguageBarProps> = ({
   targetLanguages,
   selectedLanguage,
-  setSelectedLanguage,
 }) => {
+  const { name, version } = usePackageState();
```

Comment:
I wasn't sure if there's any advantage/disadvantage retrieving state here or passing package state from the parent component (`LanguageSelection`). LMK if either is better

Reply:
I think this is fine 👍